pkg/cluster/install.go

```diff
@@ -301,20 +301,25 @@ func (m *manager) Install(ctx context.Context) error {
 			steps.Action(m.configureIngressCertificate),
 			steps.Condition(m.ingressControllerReady, 30*time.Minute, true),
 			steps.Action(m.configureDefaultStorageClass),
-			steps.Action(m.finishInstallation),
```

Comment:
Hey Spencer, thanks for this. Can you help me understand why we're removing the finishInstallation call here when the function still exists on line 366

Reply:
After further review, I see it on line 311 now as a subsequent step. Thanks!